pkg/micro-ecc/0001-Add-RIOT-Makefile.patch

```diff
@@ -0,0 +1,23 @@
+From d6c1cb4244c72a8fa7d5a8550c8193700cd0de03 Mon Sep 17 00:00:00 2001
+From: Frank <frank>
```

Comment:
In other files i provide my name an mail address. If you want to integrate it today please add "Frank Holtz <frank-riot2015@holtznet.de>". I cannot change any files today.